src/components/create-appointment-fourth/caregiver-drawer/CreateAppointmentFourthDrawer.tsx

```diff
@@ -0,0 +1,209 @@
+import { TabContext } from '@mui/lab';
+import { Box, IconButton, Stack } from '@mui/material';
+import { SyntheticEvent, useState } from 'react';
+import ArrowBackFilled from 'src/assets/icons/ArrowBackFilled';
+import FreeCancellation from 'src/assets/icons/FreeCancellation';
+import MonetizationOn from 'src/assets/icons/MonetizationOn';
+import { BIG_CAREGIVER_AVATAR_SIZE } from 'src/components/create-appointment-fourth/constants';
+import {
+  formatWorkExperienceDateRange,
+  formatWorkExperienceDateRangeTenure,
+  getMockCaregiverAvatar,
+} from 'src/components/create-appointment-fourth/helpers';
+import Drawer from 'src/components/reusable/drawer/Drawer';
+import {
+  DrawerBody,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerTitle,
+} from 'src/components/reusable/drawer/styles';
+import { useLocales } from 'src/locales';
+import { appointmentCreationApi } from 'src/redux/api/appointmentCreationAPI';
+import { setSelectedCaregiver } from 'src/redux/slices/caregiverSlice';
+import { useAppDispatch } from 'src/redux/store';
+
+import {
+  BookButton,
+  DrawerAvatar,
+  DrawerDateTenure,
+  DrawerItem,
+  DrawerStats,
+  DrawerTextHeading,
+  DrawerTextTitle,
+  DrawerTextValue,
+  StyledLink,
+  StyledTab,
+  StyledTabPanel,
+  StyledTabs,
+  StyledUnorderedList,
+  StyledVideo,
+} from './styles';
+
+interface CreateAppointmentFourthDrawerProps {
+  open: boolean;
+  onClose: () => void;
+  selectedCaregiverId: string;
+}
+
+const FIRST_SELECTED_TAB = '1';
```

Comment:
could you please put constants in a separate file?

Reply:
fixed